_posts/2021-02-11-asking-for-help.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: epic
+title: "Why Asking for Help Strengthens Engineering Teams"
+date: 2021-02-11
+categories: [team, culture, community, learning, teaching]
+author: anna
+canonical: https://medium.com/swlh/why-asking-for-help-strengthens-engineering-teams-6b1eadc86936
+---
+
+I know that for many developers, especially those early in their careers, asking for help can be intimidating. I
+often fear wasting someone’s time or exposing myself as less skilled or smart than my team initially thought.
+
+In my first month as a software engineer at Artsy (and barely six months into life as an engineer after
+[transitioning from a career in communications](https://medium.com/swlh/how-losing-my-job-helped-me-find-my-way-back-8c8f86552acc)),
+I was struggling through a ticket assigned to me as a “good first issue.” (The team estimated the task to be
+straightforward enough for someone new to the team.) After a few hours stumbling between the ticket, my code, and
+Google, I made very little progress.
+
+<!-- more -->
+
+Early and often, my fellow engineers at Artsy encourage me to ask for help when I hit roadblocks. Even with the
+support explicitly offered to me, I was shaky. The extra 10 seconds it took to type out my question in Slack was
+just enough time for me to chicken out.
+
+Opening up with my team members about my concerns, I learned that asking for help is not just for me but ultimately
+benefits my team. Sharing their insights below, I hope other engineers, early-career or not, feel empowered to ask
+questions, stay curious, and prioritize learning for themselves and their team — all hallmarks of great engineers.
+
+##Why It’s So Hard to Ask For Help
```

Comment:
Here and throughout I think you need a space after the `#`s to get the heading formatting to work?

Reply:
OOH great catch will update!

Reply:
also thank you @anandaroop!